packages/rum/test/parentContexts.spec.ts

```diff
@@ -17,6 +17,10 @@ describe('parentContexts', () => {
   const FAKE_ID = 'fake'
   const startTime = 10
 
+  function createViewCreatedEvent(partialViewCreatedEvent: Partial<ViewCreatedEvent> = {}): ViewCreatedEvent {
```

Comment:
Probably just a NIT but I think `ViewCreationEvent` would be easier to understand especially with this introduced function `createViewCreatedEvent`.

Can we also rename it something like `startViewCreationEvent` / `buildViewCreationEvent` ?

Reply:
agree that it could be nice to do something with this function name 🙂

Reply:
For the interface name: the lifecycle event is called `VIEW_CREATED`, so naturally the event name should be called `ViewCreatedEvent`. If you think `VIEW_CREATED` should be changed to `VIEW_CREATION`, we should also rename all lifecycle event (completed/completion, mutated/mutation etc.)

For the function name, I don't mind changing it to `buildViewCreatedEvent`.